docs/developing.rst

```diff
@@ -2149,12 +2149,21 @@ An example of the second approach is
     }
   }
 
-Adding a wavefunction
+Wavefunction
 ---------------------
 
-The total wavefunction is stored in ``TrialWaveFunction`` as a product
-of all the components. Each component derives from
-``WaveFunctionComponent``. The code contains an example of a
+A full ``TrialWaveFunction`` is formulated as a product
+of all the components. Each component derives from ``WaveFunctionComponent``.
+
+.. math::
+     \psi = \prod_c {\tilde \psi_c}
+
+QMCPACK doesn't directly use the product form but mostly use the log of the wavefunction which
+fits the formulation of QMC algorithms very well. It also offers a numerical advantage on computers.
+The log value grows linearly instead of exponentioally with respect to the electron counts
```

Comment:
The advantage of using the log vs. the value should be that the log doesn't exceed the range of double precision, where the value potentially could (??)  I'm not sure scaling wrt electron count matters, except that larger systems are more likely to suffer overflow/underflow on the value.

Reply:
by scaling I meant overflow/underflow on the value. Let me update.

Reply:
Fixed.